### DISCUSSION TITLE
gatsby-theme-cara : Module build failed with Unexpected token

### DISCUSSION THREAD

**Discussion by eniger31** [gatsbyjs/gatsby#36579] [Help]
Hi, I have a problem on  @LekoArts/gatsby-theme-cara. Any ideas? 😅    .babelrc created & npm install --save-dev babel-preset-gatsby . Thanks 
![Capture d’écran 2022-09-08 à 16 51 45](https://user-images.githubusercontent.com/68066790/189311937-6d4638b0-8737-4b38-a0cc-36af382502f3.png)

**Comment by marvinjude** [comment]
Hi @eniger31,

Please create an issue in https://github.com/LekoArts/gatsby-themes
